examples/spinner/main.ml

```diff
@@ -0,0 +1,44 @@
+open Minttea
+open Leaves
+
+let mint fmt = Spices.(default |> fg (color "#77e5b7") |> build) fmt
+
+type s = { spinners : Sprite.t list }
+
+let initial_model =
+  let now = Ptime_clock.now () in
+  {
+    spinners =
+      Spinner.
+        [
+          { sprite = line; frame = 0; last_frame = now };
```

Comment:
**question**. wouldn't all sprites normally start on frame 0 and with `last_frame` now? 
 Feels redundant to have to add all this stuff here instead.

Reply:
yes, makes sense to have a `Sprite.init` function for this